src/fsharp/service/IncrementalBuild.fs

```diff
@@ -1203,6 +258,44 @@ type RawFSharpAssemblyDataBackedByLanguageService (tcConfig, tcGlobals, tcState:
         member __.HasAnyFSharpSignatureDataAttribute =  true
         member __.HasMatchingFSharpSignatureDataAttribute _ilg = true
 
+module IncrementalBuild =
+
+    let mutable injectCancellationFault = false
+    let LocallyInjectCancellationFault() = 
```

Comment:
Can this be lower-case to match the F# style guide since we're not just fitting something in to the existing Incremental Builder (and its style) anymore?

Reply:
Yea this can be lowercase.